src/App.js

```diff
@@ -1,15 +1,99 @@
 import React from "react";
-import logo from "./logo.svg";
-import "./App.css";
+
+import NavBar from "./components/navbar";
+import HexGallery from "./components/hexGallery";
+import HeroBanner from "./components/heroBanner";
+import RedBanner from "./components/redBanner";
+import AboutUs from "./components/aboutUs";
+import Footer from "./components/footer";
+import CodePurposeFooter from "./components/codePurposeFooter";
+
+import cabinets from "./images/cabinets.jpg";
+import concrete from "./images/concrete.jpg";
+import carpentryFinish from "./images/carpentryFinishWork.jpeg";
+import flooring from "./images/flooring.jpg";
+import homeMaintenace from "./images/homeMaintenance.jpg";
+import homeRemodeling from "./images/homeRemodeling.jpg";
+import lighting from "./images/lighting.jpg";
+import railing from "./images/railing.jpg";
+import roughCarpentry from "./images/roughCarpentry.jpg";
+import shelving from "./images/shelving.jpg";
+import stairCases from "./images/stairCases.jpg";
+import heroBanner from "./images/heroBanner.jpg";
+
+const title = { name: "DRY CREEK INDUSTRIES", subTitle: "GENERAL CONTRACTING" };
+
+const links = [
+  { name: "Home", path: "home" },
+  { name: "Gallery", path: "gallery" },
+  { name: "Contact Us", path: "contactUs" }
+];
+
+const rowOne = [
```

Comment:
I know this visually is the first row in the UI, but what do you think about giving these a title that tell you more about what they represent?  Also, couldn't we use an array of arrays instead? and simply pass that through. Also, should the Hex component just own these values? What is the purpose of passing them through?

Reply:
I converted this to an array of arrays and renamed to galleryImages. I moved the values to the HexGallery component. I was trying to keep all the web content in one place.